aws-apigateway/index.ts

```diff
@@ -18,7 +18,7 @@ import { Provider } from "./provider";
 import { parse } from "yaml";
 
 function main(args: string[]) {
-  const file = readFileSync(require.resolve("./schema.yaml"), "utf8");
+  const file = readFileSync(require.resolve("./schema.json"), "utf8");
```

Comment:
Does `parse(file)` understand JSON vs YAML format automatically?

Reply:
In this instance, yes. CI tests passes, and I was able to validate this provider built locally as well.